addons/src/ao_markword.c

```diff
@@ -78,6 +79,13 @@ static void ao_mark_word_set_property(GObject *object, guint prop_id,
 	{
 		case PROP_ENABLE_MARKWORD:
 			priv->enable_markword = g_value_get_boolean(value);
+			/* if the plugin is loaded while Geany is already running, we need to connect the
+			 * button press signal for open documents, if Geany is just booting,
+			 * it happens automatically */
+			if (main_is_realized())
```

Comment:
shouldn't this check for `priv->enable_markword` too, not to do that on deactivation as well?

Reply:
Uhhh, of course, thanks. Stupid me. Fixed in 71e26d8f69092ddaa2da305bb3cee4f76378a8b5.